### ISSUE TITLE
Component Property of "Elevation"

### ISSUE THREAD

**Issue by danielbruns93** [siemens/ix#631]
It is very time consuming changing the elevation levels of a component manually. We would appreciate, if you could ad the property "elevation" to all associated components where an elevation level (defined by Brandville) can be applied (espacially we need it for:
- Blind
- Cards
- Date picker
- Date time picker
- Dropdown
- Empty state
- Event list
- Group
- HTML table
- KPI
- Message bar
- Modal
- Tabs
- Tile
- Time picker
- toast
- Tooltip
- Tree
- Upload

Many thanks!

**Comment by silviowolf** [comment]
Hi @danielbruns93,

the elevation principle is solved differently in iX for technical/historical reasons, and this also makes the handling easier. We use colors with opacity to achieve the stacking look. So in most cases you don’t need to care about the level.

Components that need to stand out from the background use a background color with alpha. Same to the hover, active and selected states. All these colors use opacity to work on different background colors. The only exception is currently the "Tile" with a solid color. But this will also be solved soon. 

Other compoonents like Modals, Tooltip, Datepicker, Toast are overlaying components. They will stand out from the background due to its dropshadow. Item based components like Tree or tabs don’t have a background. Their hover active and selected states work as mentioned above.

Long story short, we will not introduce the elevation levels for technical and compatibility reasons. This decision is aligned with Brand. The level principle is solved more or less implicitly in iX.

![image](https://github.com/siemens/ix/assets/115155312/54bf5d2a-501f-4c30-9bf5-ad5fed3a359e)

**Comment by danielbruns93** [comment]
All right. That sounds great :)
